### PR TITLE
Update tutorial for v0.40.0

### PR DESCRIPTION
**Pull request by jtlan**

### CODE REVIEW

File: tutorials/index.md [hunk 1]
```diff
@@ -80,14 +80,14 @@ The following is the html that you'll need to run Plottable.
   <head>
     <title>Plottable Tutorial 1: A Basic Chart</title>
 
-    <!-- use the Plottable css -->
-    <link rel="stylesheet" type="text/css" href="http://palantir.github.io/plottable/plottable.css" />
+    <!-- use the Plottable css (located one level up) -->
+    <link rel="stylesheet" type="text/css" href="../plottable.css" />
```
Comment (jtlan) [inline tutorials/index.md]: The other tutorials all use this reference though. Maybe it should just say
```HTML
<!-- use the Plottable css -->
```

Comment (bluong) [inline tutorials/index.md]: Thinking about it and to keep consistency / cleanliness, I think I'm ok with the way it is right now.  This is under expectation that the users have enough html / js experience to know that href needs to point at the file, which I think is basic knowledge.

**Comment by bluong** [comment]
Please remember to update the homepage's version number

**Comment by jtlan** [comment]
Wrong target branch, re-creating.

**Pull request closed without merging**